lametro/views.py

```diff
@@ -103,6 +106,29 @@ def extra_context(self):
 
         return extra
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        if "invalid_form" in kwargs:
+            context["alert_form"] = kwargs.get("invalid_form")
+        else:
+            context["alert_form"] = AlertForm()
+
+        return context
+
+    def post(self, request, *args, **kwargs):
+        form = AlertForm(request.POST)
+        description_content = request.POST.get("description")
+
+        if description_content.isspace():
```

Comment:
What about handling this validation on the form model a la https://docs.djangoproject.com/en/4.2/ref/forms/validation/#cleaning-a-specific-field-attribute? Then your view should Just Work™️ without having to do this manual handling.

Reply:
Ah gotcha, sounds good! In order to get that field specific validation to work, I changed the `description` field on the model to be optional, otherwise django would just run it's own validation and spit out a generic "This field is required" error message.

"Just Work™️" is something I'm going to carry with me for the rest of my career